docs/design/README.md

```diff
@@ -36,6 +36,11 @@ Generally, design docs are on Google docs:
 These docs may not be design docs in themselves, but seek to influence design
 decisions
 
+* [Mutation
```

Comment:
nit: should this just be one line?

Reply:
Arg, I thought it was just wrapping weird. Fixed. Thanks!